tools/sgen/input/FluentUIStyle.yml

```diff
@@ -120,6 +120,9 @@ Typography:
 Border:
   radius: [ none: 0pt, small: 2pt, medium: 4pt, large: 8pt, xlarge: 12pt ]
   size: [ none: 0pt, thin: 1pt, thick: 2pt, thicker: 4pt, thickest: 6pt ]
+  
+Cell:
+  height: [ oneLine: 48pt, twoLines: 64pt, threeLines: 84pt ]
```

Comment:
This doesn't seem necessary since it's exclusive to the List cells.
I'd just put the values directly in the AP_MSFListTokens:

AP_MSFListTokens:
  cellHeight: [ oneLine: 48pt, twoLines: 64pt, threeLines: 84pt ]